packages/ibm-products/src/components/Datagrid/Datagrid.test.js

```diff
@@ -2337,10 +2348,34 @@ describe('batch action testing', () => {
         expect(batchAction.label).toEqual(optionsText[index]);
       });
 
-      //coverage
-      fireEvent.click(options[0]);
-      fireEvent.click(cancelButton);
-      fireEvent.click(selectAllButton);
+      const checkMenuItem = async (
+        remainingBatchIndex,
+        clickHandlerFn,
+        initiateMenuOpen
+      ) => {
+        if (initiateMenuOpen) {
+          await click(menuButton);
+        }
+        const displayedMenuElement = document.querySelector(
```

Comment:
The only way I've seemed to get this to work correctly is to use `document.querySelector`, because the `menu` is rendered via a React portal which I believe inserts it outside of the `container`.